### PR TITLE
simplify GRASS_EXECUTABLE handling in package.sh

### PR DESCRIPTION
**Pull request by landam**

### CODE REVIEW

File: mswindows/osgeo4w/package.sh [hunk 1]
```diff
@@ -113,11 +113,7 @@ read PATCH <&3
 export VERSION=${MAJOR}.${MINOR}.${PATCH}
 export POSTFIX=${MAJOR}${MINOR}
 
-if [[ "$PATCH" == *svn* ]] ; then
```
Comment (hellik) [inline mswindows/osgeo4w/package.sh]: so we omit the indication that it's a dev version?

Comment (landam) [reply mswindows/osgeo4w/package.sh]: currently the indication of dev version (related to executable is not working anyway). I don't see any strong reason to indicated it anyway. `grassXY` seems to be enough.

Comment (landam) [reply mswindows/osgeo4w/package.sh]: similar changes are required by `GRASS-Installer.nsi.tmpl`, see eg.

```
36:        !define GRASS_COMMAND "grass@GRASS_VERSION_MAJOR@@GRASS_VERSION_MINOR@svn"
```
But it's for another PR.

Comment (landam) [reply mswindows/osgeo4w/package.sh]: @hellik Do you agree to strip 'svn/dev' postfix from executable (package name or so)? BTW, GRASS dev version is already installed into `apps/grass/grass79` and not into `apps/grass/grass79dev`

Comment (hellik) [reply mswindows/osgeo4w/package.sh]: lets strip svn/dev from executable/package name.

maybe it's better to indicate more visible for the user on the startup wizzard that it's a dev version?

Comment (landam) [reply mswindows/osgeo4w/package.sh]: yes, let's create a new PR for this purpose

**Pull request merged**